frameworks/digital-outcomes-and-specialists-5/questions/services/anonymousRecruitment.yml

```
@@ -8,4 +9,6 @@ depends:
 
 validations:
   - name: answer_required
-    message: 'You need to answer this question.'
+    message: >
+      Select yes if you can provide recuit participants without revealing
```

```suggestion
      Select if you can recruit participants without revealing
```

^ typo, and I think the 'yes' is superfluous (compare to https://github.com/alphagov/digitalmarketplace-frameworks/blob/0cf756cfe473b7c8fd7b98bd44fa5197a8cbd19b/frameworks/g-cloud-12/questions/services/serviceAddOnType.yml#L25) (and also other 'Select yes if...')

---

It's based off the [Design System guidance on error messages for radios](https://design-system.service.gov.uk/components/radios/).

Again I'm happy to change it, if Nora thinks it's a good idea, but in general we should be sticking to the Design System patterns where possible.

---

OK. That makes sense. I clearly need to get better acquainted with the Design System guidance - I'm far more familiar with the GOV.UK guidance, which can be subtly different.

Was this intentional, though?

```suggestion
      Select yes if you can recruit participants without revealing
```

---

> Was this intentional, though?

Nope, good spot :D